### ISSUE TITLE
Content request: add to website: what a license provides.

### ISSUE THREAD

**Issue by nacno-toporiame** [careteditor/caret#641]
This is not mentioned on the website of this product, and it seems that Caret can run without a license with no problems.
Is it a time-limiting trial or limited feature trial?
I suggest that you guys mention that on `caret.io`

**Comment by erusev** [comment]
I agree, we're working on a documentation section for the website that should cover this, thanks for the feedback!

**Comment by nacno-toporiame** [comment]
@erusev Could you explain some of that here?

**Comment by erusev** [comment]
Sure, the non-licensed version is the same as the licensed version and it works the same way as the licensed version except that it will occasionally interrupt you to prompt you to purchase a license.

I hope that this makes things a bit clearer.

**Comment by craigbass76** [comment]
Can I have multiple instances installed with one license? My employer gave me a Mac, but I work at home and would rather use my own Linux box. I'll only be running one instance at a time, so can I just use the license they give me on my own PC after I've also installed it on the Mac?

**Comment by astoilkov** [comment]
@craigbass76 Yes. You can do that. Licenses are per user not per computer. Wish you happy time using Caret.

**Comment by craigbass76** [comment]
Awesome
